content/200-concepts/100-components/01-prisma-schema/07-names-in-underlying-database.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Names in the underlying database'
+metaDescription: ''
+tocDepth: 2
+---
+
+<TopBlock>
+
+The [Prisma schema](./) includes mechanisms that allow you to define names of certain database objects. You can:
+
+- [Map model and field names to different collection/table and field/column names in the underlying database](#mapping-collectiontable-and-fieldcolumn-names)
+- [Define constraint and index names](#constraint-and-index-names-preview) (Preview)
+
+</TopBlock>
+
+## Mapping collection/table and field/column names
+
+Prisma model [naming conventions (singular form, PascalCase)](../../../reference/api-reference/prisma-schema-reference#naming-conventions) <span class="api"></span> do not always match table names in the database. A common approach for naming tables/collections in databases is to use plural form and [snake_case](https://en.wikipedia.org/wiki/Snake_case) notation.
```

Comment:
Opinion follows:
I dislike this explanation that puts this onto the naming conventions. In reality users often just want soemthing different in their Prisma Client and schema vs. what they have in the database. (which might very well be because they want to follow some naming conventions). Formulating it like that puts it to much on the Prisma naming conventions though.

Reply:
rephrased it.